src/networkd.c

```diff
@@ -475,6 +476,15 @@ write_network_file(net_definition* def, const char* rootdir, const char* path)
         dhcp_overrides combined_dhcp_overrides;
         combine_dhcp_overrides(def, &combined_dhcp_overrides);
 
+        /* Only set MTU from DHCP if use-mtu dhcp-override is not false. */
+        if (!combined_dhcp_overrides.use_mtu) {
+            /* isc-dhcp dhclient compatible UseMTU, networkd default is to
+             * not accept MTU, which breaks clouds */
```

Comment:
Do you have a bug reference for the "breaks clouds" comment?

Reply:
This is an existing comment, I retained it to keep the context on why UseMTU defaults to true.